ImageKit/ImageKit.swift

```diff
@@ -6,27 +6,27 @@
 //
 
 import Foundation
-public let API_VERSION: String = Bundle(identifier: "org.cocoapods.ImageKitIO")?.infoDictionary!["CFBundleShortVersionString"] as? String ?? "Unknown"
```

Comment:
we have completely dropped API_VERSION from the generated url, @manu4543 should we have a flag where it is conditionally set? like by default it is added but you can configure it to be off

Reply:
it should never be set now we dropped it in all places. IIRC, we removed it in android too @AniVerma17

Reply:
> it should never be set now we dropped it in all places. IIRC, we removed it in android too @AniVerma17

Yes, this parameter was completely removed with no option to toggle by the consumer app.